### PR TITLE
Tin 6

### PR DESCRIPTION
**Pull request by a40o**
User edit

### CODE REVIEW

File: src/main/java/com/volasoftware/tinder/controller/EditingController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+public class EditingController {
+    private final UserService userService;
+
+    public EditingController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PutMapping("/api/v1/users/profile")
+    @SecurityRequirement(name = "bearer")
+    public ResponseEntity editUser(@Valid @RequestBody UserDto userDto) throws MessagingException, IOException {
```
Comment (todornikolov1) [inline src/main/java/com/volasoftware/tinder/controller/EditingController.java]: Move this API method to the UserController. Please, create each controller based by domain (aka business entity), adding all operations (CRUD and others) in it

File: src/main/java/com/volasoftware/tinder/exception/UserDoesNotExistException.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
-package com.volasoftware.tinder.exception;public class UserDoesNotExistException extends RuntimeException{
+package com.volasoftware.tinder.exception;
 
+public class UserDoesNotExistException extends RuntimeException{
```
Comment (todornikolov1) [inline src/main/java/com/volasoftware/tinder/exception/UserDoesNotExistException.java:3]: How this is different from UserNotFoundException as logic?

File: src/main/java/com/volasoftware/tinder/services/UserService.java [hunk 3]
```diff
@@ -80,6 +78,16 @@ public void registerUser(UserDto userDto) throws MessagingException, IOException
                     verificationService.setVerificationLink(String.valueOf(token)));
     }
 
+    public void editUser(UserDto input) throws MessagingException, IOException{
+
```
Comment (todornikolov1) [inline src/main/java/com/volasoftware/tinder/services/UserService.java]: remove this empty space

File: src/main/java/com/volasoftware/tinder/exception/UserDoesNotExistException.java (hunk 2 above)
Comment (a40o) [reply src/main/java/com/volasoftware/tinder/exception/UserDoesNotExistException.java:3]: I just thought that I should  make new exception class even thou the cases are identical

File: src/main/java/com/volasoftware/tinder/controller/EditingController.java (hunk 1 above)
Comment (a40o) [reply src/main/java/com/volasoftware/tinder/controller/EditingController.java]: All the controller are unified

**Review by todornikolov1 — APPROVED** [review]

**Pull request merged**